persistence/src/main/java/edu/softserve/zoo/persistence/provider/impl/HibernatePersistenceProvider.java

```diff
@@ -133,15 +133,15 @@ public boolean delete(Long id, Class type) {
      */
     @Override
     @Transactional(propagation = Propagation.MANDATORY, readOnly = true)
-    public List<T> find(Specification<T> specification) {
+    public <K> List<K> find(Specification<T> specification) {
```

Comment:
![image](https://cloud.githubusercontent.com/assets/13787714/15857031/cfaf7db4-2cc1-11e6-861d-df63545e0f7f.png)
I suppose you won't get a raw list